config/sync/config_ignore.settings.yml

```diff
@@ -2,13 +2,6 @@ _core:
   default_config_hash: v7E8C8wJWeAW2BGohMNY1tZSBc4bexM6O62tGxecTfE
 mode: simple
 ignored_config_entities:
-  - 'core.extension:theme'
-  - 'system.theme:default'
-  - '~block.block.claro_*'
-  - '~block.block.stanford_profile_admin_*'
-  - '~block.block.stanford_basic_*'
-  - '~block.block.cardinal_service_*'
-  - '~block.block.minimally_branded_subtheme_*'
```

Comment:
We shouldn't be removing these

Reply:
@pookmish I confirmed on my local these are in there. In fact, it's in there twice. Not sure what's going on with that. 

Maybe I should just pull the 11.x version and upload it. Pretty sure I didn't edited this.